aziot/src/internal/check/checks/well_formed_configs.rs

```diff
@@ -0,0 +1,179 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+use super::prelude::*;
```

Comment:
Please no wildcard imports, even from our own preludes.

Reply:
Uhh, I'm not sure I agree with you there. The whole point of the prelude is to wildcard import it, otherwise what's the point? If you're already importing types from the prelude manually, you might as well import the types using their fully-qualified paths to begin with.

Given that this prelude is private and can only be used by individual check implementations - all of which follow a rigid and matching module and import structure - I think using a prelude here is an appropriate way to cut down on noise in the implementations.

Reply:
+1 on using `tokio::prelude::*`. The tokio docs suggest manually importing it this way too. On a sidenote, tokio 1.0 no longer has the prelude module I believe - so this will eventually vanish?